packages/terra-table/src/SelectableTableRows.jsx

```diff
@@ -31,24 +35,77 @@ const propTypes = {
 const defaultProps = {
   disableUnselectedRows: false,
   onChange: undefined,
+  onBlur: undefined,
   selectedIndexes: [],
 };
 
+const rowsSelector = 'row-group';
+
 class SelectableTableRows extends React.Component {
   constructor(props) {
     super(props);
     this.handleOnChange = this.handleOnChange.bind(this);
+    this.handleOnBlur = this.handleOnBlur.bind(this);
+  }
+
+
+  componentDidMount() {
+    // Adds ARIA-Label for Pre-selected rows
+    const rowGroup = document.querySelector(`#${rowsSelector}`);
+    for (let rowIndex = 0; rowIndex < rowGroup.children.length; rowIndex += 1) {
```

Comment:
I'd calculate the length before the loop and avoid calculating it during each iteration unless doing it this way gives an advantage.

Reply:
i was not sure that it calculates the length at each iteration. assumed that it may be called only one time during initial loop. 
added const to hold length: [here](https://github.com/cerner/terra-core/blob/cc6ee3fad6bea01cc24bb5ce812e6483291d781d/packages/terra-table/src/SelectableTableRows.jsx#L68)